io/js/src/test/scala/fs2/io/NodeJSCompressionSuite.scala

```diff
@@ -40,21 +45,191 @@ class NodeJSCompressionSuite extends CompressionSuite {
     options.strategy = strategy
     val out =
       if (nowrap)
-        facade.zlib.gzipSync(in, options)
+        facade.zlib.deflateRawSync(in, options)
       else
         facade.zlib.deflateSync(in, options)
     Chunk.uint8Array(out).toArray
   }
 
-  override def inflateStream(b: Array[Byte], nowrap: Boolean): Array[Byte] = {
+  def inflateStream(b: Array[Byte], nowrap: Boolean): Array[Byte] = {
     val in = Chunk.array(b).toUint8Array
     val options = new facade.zlib.Options {}
     val out =
       if (nowrap)
-        facade.zlib.gunzipSync(in, options)
+        facade.zlib.inflateRawSync(in, options)
       else
         facade.zlib.inflateSync(in, options)
     Chunk.uint8Array(out).toArray
   }
 
+  test("gzip |> gunzip ~= id") {
```

Comment:
Frustratingly, these tests are cloned from the JVM suite except modified to remove the optional headers unsupported on Node.js. The duplication is unfortunate but I didn't see a clean way to share them.